docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -0,0 +1,388 @@
+= Using the Generic MapLoader
+:description: With the xref:working-with-external-data.adoc#options[generic MapLoader], you can configure a map to cache data from an external data store. This topic includes an example of how to configure a map with a generic MapLoader that connects to a MySQL database.
+:page-beta: true
+
+{description}
+
+For a list of all supported databases, see the xref:external-data-stores:external-data-stores.adoc#factories[built-in data store factories].
+
+== Before you Begin
+
+You need a xref:external-data-stores:external-data-stores.adoc[data store connection] that's configured on all cluster members.
+
+== Quickstart Configuration
+
+This example shows a basic map configuration for an external datastore called `my-mysql-database`. See xref:data-structures:map.adoc[] for the details of other properties that you include in your map configuration.
+
+[tabs] 
+==== 
+XML:: 
++ 
+-- 
+[source,xml]
+----
+<hazelcast>
+    ...
+    <map name="default">
```

Comment:
I am not sure if creating a **default** map with a mapstore is a good example. Because then all the maps are going to have the same mapstore. Mapstores are supposed to be used with a unique map in my opinion

Reply:
Updated in all instances. Can you check?